static/docs/user-guide/cache-file-linking.md

```diff
@@ -41,61 +40,73 @@ system.
 There are pros and cons to the 3 supported link types: Hard links, Soft or
 Symbolic links, and Reflinks\* in more recent systems. While reflinks bring all
 the benefits and none of the worries, they're not commonly supported in most
-platforms yet. Hard/soft links also optimize speed and space in the file system,
-but carry the risk of breaking your workflow, since updating tracked files in
-the workspace causes data corruption. These 2 link types thus require using
+platforms yet. Hard/soft links optimize speed and space in the file system, but
+carry the risk of breaking your workflow since updating hard/sym-linked files in
+the workspace causes cache corruption. These 2 link types thus require using
 cache protected mode (see the `cache.protected` config option in
-`dvc config cache`). Finally, a 4th "linking" option is to actually `copy` files
-from/to the cache, which is safe but quite inefficient for large files (not
-recommended for GBs of data).
+`dvc config cache`). Finally, a 4th "linking" option is to actually copy files
+from/to the cache, which is safe but inefficient, especially for large files
+(several GBs or more data).
 
-Each file linking option is further detailed below, in order of efficiency:
+File link type benefits summary:
+
+| `cache.type` | speed | space | safe |
+|---|---|---|---|---|
+| `reflink` | x | x | x |
+| `hardlink` | x | x | |
+| `symlink` | x | x | |
+| `copy` | | | x |
+
```

Comment:
FYI added the table above @shcheklein

Reply:
👍 can we rename the `safe` column somehow? it feels like hardlinks and symlinks are really bad - unsafe :) while they just need extra measures.

Reply:
How about "editable" ?